package.json

```diff
@@ -52,7 +52,7 @@
     "eslint-scope": "^5.0.0",
     "eslint-utils": "^1.4.2",
     "eslint-visitor-keys": "^1.1.0",
-    "espree": "^6.1.1",
+    "espree": "github:eslint/espree#use-parser-acorn",
```

Comment:
Does this mean that we no longer have a way to control version of espree that's being pulled in? It will always pull it directly from github bypassing NPM?

Reply:
I'm assuming this is temporary so that it can be tested out while the PR is still a work in progress, but I could be wrong.

Reply:
This is a temporary fix to confirm that this change fixes the problem 😄 . After `espree` has been released, it will be fixed to `^6.2.0` or something like. This is the reason that this PR has "do not merge" label.